buildkite/src/Jobs/Lint/Merge.dhall

```diff
@@ -43,16 +43,6 @@ Pipeline.build
           , docker = Some Docker::{
               image = (../../Constants/ContainerImages.dhall).toolchainBase
             }
-        },
-      Command.build
```

Comment:
Can you make it soft-fail instead? We can use the branch protection rules to ensure that this only applies on `compatible` and then it shouldn't matter where this job ends up running